odl/contrib/mrc/mrc.py

```diff
@@ -20,7 +20,7 @@
 import struct
 import warnings
 
-from odl.tomo.data.uncompr_bin import (
+from .uncompr_bin import (
```

Comment:
Oh no, nothing specific. Probably a work-in-progress change while moving stuff around.